pkg/credentials/credentials.go

```diff
@@ -0,0 +1,35 @@
+package credentials
+
+import (
+	"path/filepath"
+)
+
+type TLSCredentials struct {
+	credentialsPath string
+}
+
+func NewTLSCredentials(path string) TLSCredentials {
```

Comment:
nit : comments ..applicable to all public methods/structs..

Reply:
This has comments.